src/components/metadata/metadata.test.js

```diff
@@ -93,70 +97,59 @@ describe('MetaData', () => {
     });
 
     it('shows the node type as an icon', () => {
-      const wrapper = mount({ nodeId: salmonTaskNodeId });
+      const wrapper = mount({
+        nodeId: salmonTaskNodeId,
+        mockMetadata: node_task,
```

Comment:
see my above comment - this change would not be needed if we take out the mock action in the mount function, as here we are testing something that does not need mock data and hence this proposed change is redundant